python/tvm/relay/op/contrib/arm_compute_lib.py

```diff
@@ -337,3 +354,17 @@ def global_avg_pool2d(attrs, args):
     if attrs.layout != "NHWC":
         return False
     return True
+
+
+@tvm.ir.register_op_attr("add", "target.arm_compute_lib")
+@tvm.ir.register_op_attr("qnn.add", "target.arm_compute_lib")
+def add(attrs, args):
+    """Check if the external ACL codegen for add should be used."""
+    for typ in [args[0].checked_type, args[1].checked_type]:
+        if typ.dtype not in ["float32", "uint8"]:
```

Comment:
I think it would be better to write add and qnn.add separately. This way we can state add must have float32 input tensors and qnn.add must have uint8 inputs. Currently we allow a mixture of types.

Reply:
1. Yep. The pattern is not needed. Removed. 
2. Yep. Has been split